lib/tika.rb

```diff
@@ -16,6 +16,12 @@ def self.get_recursive_metadata(file)
     JSON.parse(body)
   end
 
+  def self.tika_version
+    response = client.get('/version')
+
+    response.to_s
+  end
```

Comment:
While adding a test for this method I realized there should at least be some rudimentary error handling for this method (in case the Tika api is unavailable and does not return a version).  I mimicked the existing Tika error used in `get_recursive_metadata` and added a test for this as well.